### PR TITLE
Arachne 2391 batch actions on admin users

### PR DESCRIPTION
**Pull request by agackovka**

### CODE REVIEW

File: src/modules/Admin/components/PortalUserList/Table/ActionsToolbar/container.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ * Company: Odysseus Data Services, Inc.
+ * Product Owner/Architecture: Gregory Klebanov
+ * Authors: Anton Gackovka
+ * Created: May 23, 2018
+ */
+
+import actions from 'actions/index';
+import { ContainerBuilder, Utils } from 'services/Utils';
+import ActionsToolbar from './presenter';
+import selectors from '../selectors';
```
Comment (johnSamilin) [inline src/modules/Admin/components/PortalUserList/Table/ActionsToolbar/container.js]: better use absolute path

File: src/modules/Admin/components/PortalUserList/Table/ActionsToolbar/container.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      [batchOperationType.DELETE]: "delete",
+      [batchOperationType.RESEND]: "render confirmation email to",
+      [batchOperationType.CONFIRM]: "toggle confirmed flag for",
+    };
+    
+    return {
+      ...stateProps,
+      ...ownProps,
+      ...dispatchProps,
+      batch: (operationType) => {
```
Comment (johnSamilin) [inline src/modules/Admin/components/PortalUserList/Table/ActionsToolbar/container.js]: I'd use async/await

File: src/modules/Admin/components/PortalUserList/Table/ActionsToolbar/presenter.jsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+/** @augments{ Component<any, any>} */
+export default class ActionsToolbar extends Component{
+
+  
+  getButtons() {
+    
+    return [
+      {
+        onClick: console.log,
```
Comment (johnSamilin) [inline src/modules/Admin/components/PortalUserList/Table/ActionsToolbar/presenter.jsx]: ???

File: src/modules/Admin/components/PortalUserList/Table/ActionsToolbar/presenter.jsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  
+
+  render() {
+
+    const classes = new BEMHelper('admin-portal-user-list-actions-toolbar');
+    const tooltipClass = new BEMHelper('tooltip');
+    
+    return (
+        <div {...classes()}>
+          <div {...classes()}>
```
Comment (johnSamilin) [inline src/modules/Admin/components/PortalUserList/Table/ActionsToolbar/presenter.jsx]: forgot element

File: src/modules/Admin/components/PortalUserList/Table/container.js [hunk 5]
```diff
@@ -56,12 +57,14 @@ class UserListTableBuilder extends ContainerBuilder {
...
     return {
       loadList: actions.adminSettings.portalUserList.find,
-      removeUser: actions.adminSettings.portalUserList.delete,
       enableUser: actions.adminSettings.portalUserEnable.create,
       confirmEmail: actions.adminSettings.portalUserConfirmEmail.create,
+      selectUser: actions.adminSettings.portalUserListSelectedUsers.toggle,
       setSearch: actions.router.setSearch,
       search: goToPage,
       loadUsersWithCurrentQuery: (query) => actions.adminSettings.portalUserList.query({ query }),
+      loadUserIdsWithCurrentQuery: (query) => actions.adminSettings.portalUserListSelectedUsers.loadUserIds({ query }),
```
Comment (johnSamilin) [inline src/modules/Admin/components/PortalUserList/Table/container.js]: does not meet eslint rules

File: src/modules/Admin/components/PortalUserList/container.jsx [hunk 6]
```diff
@@ -70,7 +70,7 @@ class UserList extends Component {
   }
 
   render() {
-    return presenter(this.props);
+    return <List {...this.props} />;
```
Comment (johnSamilin) [inline src/modules/Admin/components/PortalUserList/container.jsx]: container should not contain jsx

File: src/modules/Admin/components/PortalUserList/presenter.jsx [hunk 7]
```diff
@@ -20,45 +20,49 @@
...
-      >
-        <Table />
-      </Grid>
-      <ModalAddUser />
-    </PageWrapper>
-  );
-}
+export default class UserList extends Component {
+
+  constructor() {
```
Comment (johnSamilin) [inline src/modules/Admin/components/PortalUserList/presenter.jsx]: useless constructor

File: src/modules/Admin/components/PortalUserList/Table/ActionsToolbar/container.js [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+      selectedUsers: selectors.getSelectedUsers(state),
+      query,
+      pathname,
+    };
+  }
+
+  getMapDispatchToProps() {
+    return {
+      batch: actions.adminSettings.portalUserList.batch,
+      loadUsersWithCurrentQuery: (query) => actions.adminSettings.portalUserList.query({ query }),
```
Comment (pavgra) [inline src/modules/Admin/components/PortalUserList/Table/ActionsToolbar/container.js]: `loadUsersWithCurrentQuery` means load with _current_ query, not with any. Such function could be created in mergeProps. But this one is just loadUsers. Same story below in several places

**Pull request merged**